koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -40,6 +40,7 @@
 from osbs.api import OSBS
 from osbs.conf import Configuration
 from osbs.exceptions import OsbsValidationException
+from osbs.utils import split_module_spec
```

Comment:
Would be worth having this catch `ImportError` to retain compatibility with older versions of osbs-client.

Reply:
Something like?

```
+try:
 +    from osbs.utils import split_module_spec
+    osbs_flatpak_support = True
+except ImportError, e:
+    osbs_flatpak_support = False

[...]

         if flatpak:
+            if not osbs_flatpak_support:
+                raise BuildError("osbs-client on koji builder doesn't have Flatpak support")
```

Reply:
Yes, like that.